stratum/hal/bin/barefoot/BUILD

```diff
@@ -80,7 +80,12 @@ stratum_cc_binary(
         "//stratum/hal/lib/barefoot:bfrt_table_manager",
         "//stratum/hal/lib/barefoot:bfrt_counter_manager",
         "@local_barefoot_bin//:bfsde",
-    ] + stratum_bf_common_deps,
+    ] + stratum_bf_common_deps + [
+        # PI needed for linking libdriver.so symbols if/when pi is enabled
+        # when building bf-drivers. This shouldn't hurt, but can be excluded
+        # when pi is removed from the SDE build options.
+        "//stratum/hal/lib/pi:pi_bf",
```

Comment:
Could this dependency added to the `:bfsde` target directly?

Reply:
Yes, this might be a better idea